### PR TITLE
Remove .lgtm.yml since this role has no python code.

### PR DESCRIPTION
**Pull request by nhosoi**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ Fixes #63
 
 ### Other Changes
 
-- changelog_to_tag action - support other than "master" for the main branch name (#66)
+- changelog_to_tag action - support other than "master" for the main branch name (#66) // wokeignore:rule=master
```
Comment (richm) [inline CHANGELOG.md:108]: this change is unrelated?

Comment (nhosoi) [reply CHANGELOG.md]: Oops, sorry. I removed the diff from the commit.

**Review by richm — APPROVED** [review]

**Comment by nhosoi** [comment]
Should I be worried about this `Integration tests / ubuntu (ubuntu-20.04)` failure? Is it just a timing issue???

ERROR! Error when getting available collection versions for community.general from default (https://galaxy.ansible.com/api/) (HTTP Code: 502, Message: Bad Gateway Code: Unknown)

**Comment by martinpitt** [comment]
@nhosoi : That was just some network glitch, retry worked.

**Comment by martinpitt** [comment]
... but this branch was out of date, so I clicked the rebase button. Tests need to run again.
